pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -131,14 +135,43 @@ def __call__(self, value, system):
                     except ValueError:
                         true_nb_of_toc = 1
 
+                    log.debug('True number of TOC pages is {}'.format(true_nb_of_toc))
```

Comment:
Maybe the expected number can also be written to the log (in case it is computed).